test/functorch/test_ops.py

```diff
@@ -1118,6 +1119,7 @@ def test():
         xfail('cummax'),
         xfail('cummin'),
         xfail('cumprod'),
+        xfail('fill'),
```

Comment:
What is going on here?

Reply:
There isn't a batching rule for `torch.fill` but since the old OpInfo only called the inplace version, the test was passing erroneously.

Reply:
In the future, please give us a head's up if you add an xfail to the functorch test suite. It is important that these tests pass.

This PR regresses functorch testing for Tensor.fill_: now, if someone modifies the code for Tensor.fill_ in an incorrect way, no one will notice.